src/app/project/project-permissions/permissions-table-rows/permissions-table-rows.component.html

```diff
@@ -1,9 +1,9 @@
-<tr *ngFor="let entry of entries"
+<tr *ngFor="let entry of entries | paginate: { id: 'table-template-pagination', itemsPerPage: paginationData.pageSize, currentPage: paginationData.currentPage, totalItems: paginationData.totalListItems }"
     class="clickable-row">
     <td data-label="User" scope="row" class="col-8">{{entry.displayName}}</td>
     <td data-label="Access" scope="row" class="col-4">
       <mat-checkbox (change)="handlePermissionCheckboxChange($event, entry)"
-                    [checked]=hasProjectPermission(entry)>
+                    [checked]="entry.projectPermissions.includes(currentProject._id)">
```

Comment:
I imagine you changed this for troubleshooting initially. Does the function work here?

Reply:
Yep the function was just an unneeded wrapper in my opinion, but I can add it back if you like.

Reply:
No biggie in this case but since it's a widely-used function in the class then adding it back could save a bit of code and keep things consistent